src/components/Card.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Card.css";
+import { Card as CardComp, Button } from "react-bootstrap";
+import { BsFillTrashFill } from "react-icons/bs";
+
+const Card = (props) => {
+  const onClickUpdateCard = () => {
+    const updateCard = {
```

Comment:
I would consider naming this `updatedCard` or `newCard`. Since `updateCard` uses the verb form, it implies that this variable is a function performing an update rather than an object holding data.